osbrain/address.py

```diff
@@ -171,6 +172,37 @@ def twin(self):
         return self.__class__(self.ZMQ_KIND_TWIN[self])
 
 
+class AgentAddressSerializer():
```

Comment:
Maybe it makes more sense to create this class inheriting from `str` (i.e.: have a look at the `AgentAddressTransport` implementation). It simplifies its creation and makes other things easier in the code later. Also, I don't thing the environment default should be loaded here (that should rather be in the Agent's implementation).